CONTRIBUTING.md

```diff
@@ -195,6 +196,46 @@ copyrights.
+    * Of course there are always exceptions.  Prefer clarity in the history over pedantically keeping under limits.
+
+### Checklist before PR creation
+
+* Run `git clang-format` and commit the changes and push before making the PR.  (There’s no real danger to the build if you forget; GitHub Actions CI will fail on the PR until `clang-format` is applied anyway.)
+* Squash commits like “apply clang-format” or “clean up”.  Squash pairs of commits that introduce a change and revert the change.  Multiple commits within a single PR are encouraged if the PR contains sub-functions that make sense to be represented individually in the Git history.
+* Create a PR with a descriptive subject, like “Fix crash in vulkaninfo” or “Add tracking for all types derived from Wrapper” and create a helpful description.  If a PR is submitted in order to share and receive comments and run CI before the PR is submitted for final approvals, mark it as Draft in Github.
+* In the case of an issue fix, put the issue number being fixed in the final commit message or at least in the PR description so the Github issue and PR are linked, and the issue is updated when the PR is merged.  It has to be a particular phrasing to match the pattern to link the PR with the issue, like “fixes #341”; see [the Github page on this.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
+    * E.g. `Fixes #341`
+
+### Checklist before PR merge
+
+* Did relevant test cases get run by hand against the PR?  e.g. Android Vulkan replay, DX12 trimmed capture, `convert`, etc - include them and the tested hardware / driver configuration in a comment in the PR.
+* Did Github Workflows CI pass? This is the "checks passed" or "checks failed" section at the bottom of the PR and represents whether the automated build in a container succeeded.  This set of “CI builds” includes no replay of capture files.
+![Github Workflows CI Screenshot](docs/img/gfxreconstruct-pr-process-02.png "Github Workflows CI Screenshot")
+* Did the PR change `gfxrecon-convert `output?
+    * If so, does the output still appear correct for changed items? Include a cut-and-paste of output before and after for review.
+    * Does the output validate correctly e.g. through the JSON tool [`jq`](https://stedolan.github.io/jq/)?
+* Has the PR received an appropriate level of review by GFXReconstruct maintainers? For small, local changes a single review by any maintainer should be sufficient, but larger, architectural ones should be directed to team members with experience of the affected areas and be signed-off by the team lead before merging.
```

Comment:
I added this item.

Reply:
I think this can be omitted.  We won't merge it if we haven't reviewed it enough internally to LunarG.  I think this is our responsibility, not the third-party PR submitter.

Reply:
Removed